trajectory_follower_wrapper/src/trajectory_follower_wrapper_node.cpp

```diff
@@ -64,12 +67,12 @@ namespace trajectory_follower_wrapper
     auto sub_qos_transient_local = rclcpp::QoS(rclcpp::KeepAll()); // A subscriber with this QoS will store all messages that it has sent on the topic
     sub_qos_transient_local.transient_local();
     // Setup subscriber
-    control_cmd_sub_ = create_subscription<autoware_auto_msgs::msg::AckermannControlCommand>("output/control_cmd", sub_qos_transient_local,
+    control_cmd_sub_ = create_subscription<autoware_auto_msgs::msg::AckermannControlCommand>("trajectory_follower/control_cmd", sub_qos_transient_local,
```

Comment:
We don't need to change the names for the topics here sine we are remapping them in the launch file